var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -26,6 +26,7 @@ class PyNumpy(PythonPackage):
         'numpy.distutils.command', 'numpy.distutils.fcompiler'
     ]
 
+    version('1.16.2', sha256='6c692e3879dde0b67a9dc78f9bfb6f61c666b4562fd8619632d7043fb5b691b0')
```

Comment:
And this? The two changes rebuild a lot of stuff. Is there a feature we cannot live without?

Reply:
Actually, tests pass with the current version. I will remove this change from this PR.